frontend/src/common/localStorage/set.ts

```diff
@@ -0,0 +1,8 @@
+export enum BOOLEAN {
+  TRUE = "true",
+  FALSE = "false",
+}
+
+export function set(key: string, value: BOOLEAN) {
+  window?.localStorage?.setItem(key, value);
```

Comment:
do we need another catch, if the browser doesn't support localStorage?

Reply:
Thanks for catching this!

Since we use optional chaining, if the browser doesn't support localStorage, it will not call `setItem()` on `window.localStorage`, which could be `undefined`, so it won't throw a runtime error and thus nothing to catch!

But thanks for catching this, because it made me realize my previous implementation of `get()` had a bug, where `catch()` block will never execute, so the function could return `undefined`, if localStorage is not supported

BEFORE
```ts
export function get(key: string) {
  try {
    return window?.localStorage?.getItem(key); // string, null, or undefined
  } catch (e) {
    return null;
  }
}
```

AFTER:
```ts
export function get(key: string): string | null {
  return window?.localStorage?.getItem(key) || null; // string or null
}

```